azext_iot/_help.py

```diff
@@ -576,6 +576,63 @@
         -m {module_name} -j ../mymodtwin.json
 """
 
+'''
+helps[
+    "iot hub state"
+] = """
+    type: group
+    short-summary: Manage the state of an IoT Hub.
+"""
+
+helps[
+    "iot hub state export"
+] = """
+    type: command
+    short-summary: Export the state of an IoT Hub to a file.
+    long-summary: The exported state will include hub configurations, edge deployments, device identities and twins, and module
+                  identities and twins.
+    examples:
+    - name: Export the state of the specified hub to the specified file.
+      text: >
+        az iot hub state export -n {iothub_name} -f {filename}
+    - name: Export the state of the specified hub to the specified file, overwriting the file contents.
+      text: >
+        az iot hub state export -n {iothub_name} -f {filename} --of
+"""
+
+helps[
+    "iot hub state import"
+] = """
+    type: command
+    short-summary: Import a Hub state from a file to an IoT Hub.
+    long-summary: The imported state will include hub configurations, edge deployments, device identities and twins, and module
+                  identities and twins.
+    examples:
+    - name: Import the state from the specified file to the specified hub.
+      text: >
+        az iot hub state import -n {iothub_name} -f {filename}
+    - name: Import the state from the default file to the specified hub, overwriting the previous state of the hub.
+      text: >
+        az iot hub state import -n {iothub_name} -f {filename} -r
```

Comment:
I think we should spell out `--replace` in this example (and remember to align on the action description,  overwrite -> replace).